test/built-ins/Promise/prototype/finally/subclass-reject-count.js

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) 2018. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+author: Jordan Harband
+description: Promise subclass finally on rejected creates the proper number of subclassed promises
+esid: sec-promise.prototype.finally
+features: [Promise.prototype.finally]
+flags: [async]
+---*/
+
+var count = 0;
+class FooPromise extends Promise {
+  constructor(resolve, reject) {
+    count++;
+    return super(resolve, reject);
+  }
+}
+
+FooPromise.reject().finally(() => {}).then($DONE).catch(() => {
+  assert.sameValue(6, count);
```

Comment:
This should be 7 -- the first 6 promises are created same as the below test and +1 for the extra `.catch()`

Reply:
aha, thanks